multimodal/tests/unittests/predictor/test_predictor.py

```diff
@@ -677,6 +678,31 @@ def test_load_ckpt():
     npt.assert_equal(predictions_prob, predictions2_prob)
 
 
+def test_fttransformer_load_ckpt():
+    download("https://automl-mm-bench.s3.amazonaws.com/ft_transformer_pretrained_ckpt/iter_2k.ckpt", "./")
+
+    dataset = ALL_DATASETS["petfinder"]
+    metric_name = dataset.metric
+
+    predictor = MultiModalPredictor(
+        label=dataset.label_columns[0],
+        problem_type=dataset.problem_type,
+        eval_metric=metric_name,
+    )
+    hyperparameters = {
+        "model.names": ["ft_transformer"],
+        "model.ft_transformer.checkpoint_name": "./iter_2k.ckpt",
```

Comment:
It would be a nice ease of use improvement if the user could specify a URL here:

instead of 

```
download(...)
"model.ft_transformer.checkpoint_name": "./iter_2k.ckpt",
```

Just do:

```
"model.ft_transformer.checkpoint_name": "https://automl-mm-bench.s3.amazonaws.com/ft_transformer_pretrained_ckpt/iter_2k.ckpt"
```

Internally we can call download and save it to some predefined directory for model weights.

For example, if this was the case then I wouldn't need to edit my benchmarking code to include the `download` call, which would simplify benchmarking a lot.

Reply:
Currently I get the following error if I try to pass the URL:

```
FileNotFoundError: [Errno 2] No such file or directory: 'https://automl-mm-bench.s3.amazonaws.com/ft_transformer_pretrained_ckpt/iter_2k.ckpt'
```

Reply:
This also would allow me to add the weights as part of the default hyperparameter configs.